### PR TITLE
Add unique IDs

### PR DESCRIPTION
**Pull request by cdheiser**
- Pull and store the GUID from the XML
  - Pull and store the UUID for all entities except OccupancyGroups

This will enable the generation of consistent unique IDs for platforms like HomeAssistant.

### CODE REVIEW

File: pylutron/__init__.py [hunk 1]
```diff
@@ -298,7 +301,8 @@ def _parse_keypad(self, keypad_xml, device_group):
                     name=keypad_xml.get('Name'),
                     keypad_type=keypad_xml.get('DeviceType'),
                     location=device_group.get('Name'),
-                    integration_id=int(keypad_xml.get('IntegrationID')))
+                    integration_id=int(keypad_xml.get('IntegrationID')),
+                    uuid=int(keypad_xml.get('UUID')))
```
Comment (thecynic) [inline pylutron/__init__.py:305]: why not store them as strings? You don't need it to be numeric as far as i can tell.

Comment (cdheiser) [reply pylutron/__init__.py]: Fair point.  Changed to just store the string UUID.  Also added a comment for the GUID.

**Comment by cdheiser** [comment]
bump

**Comment by Tuckie** [comment]
Just wanted to say thank you for this @cdheiser! I was just wanting this in HA.

**Comment by thecynic** [comment]
I'll push new pypi shortly.

**Pull request merged**